pkg/jsonmessage/jsonmessage_test.go

```diff
@@ -180,23 +180,23 @@ func TestJSONMessageDisplay(t *testing.T) {
 			Progress: &JSONProgress{Current: 1},
 		}: {
 			"",
-			fmt.Sprintf("%c[1K%c[K\rstatus       1B\r", 27, 27),
+			fmt.Sprintf("%c[2K\rstatus       1B\r", 27),
```

Comment:
I was curious why these codes were changed

Reply:
Sorry, ignore me, I realise now that this is just changing the test to match what aec does (you even said so, oops). If it isn't portable enough we'll have to patch uostreram -- but lets assume it works widely enough until someone comes up with a counter example.

Reply:
no worries. Marking this conversation as resolved.